beacon_node/store/Cargo.toml

```diff
@@ -33,6 +33,7 @@ zstd = { workspace = true }
 bls = { workspace = true }
 smallvec = { workspace = true }
 logging = { workspace = true }
+#eth2 = { workspace = true }
```

Comment:
```suggestion
```

Reply:
I can't seem to commit this suggestion, but if you commit it we can merge shortly. After this PR is back-merged to unstable to fix CI:

- https://github.com/sigp/lighthouse/pull/6671

Reply:
ah, missed that line, good catch

sorry, will allow edits in the future!